components/brave_wallet_ui/components/desktop/views/accounts/accounts.tsx

```diff
@@ -271,6 +290,34 @@ export const Accounts = () => {
           </Column>
         </>
       }
+
+      {externalAccounts.length > 0 &&
+        <>
+          <Row
+            padding='8px'
+            justifyContent='flex-start'
+          >
+            <SectionTitle>
+              {getLocale('braveWalletConnectedAccounts')}
+            </SectionTitle>
+          </Row>
+          <Column
+            fullWidth={true}
+            alignItems='flex-start'
+            margin='0px 0px 24px 0px'
+          >
+            {externalAccounts.map((account) =>
+              <AccountListItem
+                key={account.accountId.uniqueKey}
+                onClick={onSelectAccount}
+                account={account}
+                tokenBalancesRegistry={tokenBalancesRegistry}
+                spotPriceRegistry={spotPriceRegistry}
+              />
+            )}
```

Comment:
Perf: Let's just render the one item for now

Reply:
Updated